ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -0,0 +1,532 @@
+# Table of Contents <!-- omit in toc -->
```

Comment:
Required! 

A blank line after heading. Else this will throw an [MD error](https://github.com/DavidAnson/markdownlint/blob/v0.24.0/doc/Rules.md#md022)